ci-operator/step-registry/aws/provision/cco-manual-users/aws-provision-cco-manual-users-commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
This script is creating static IAM users, right? If right, suggest to rename it to `ci-operator/step-registry/aws/provision/cco-manual-users/static` folder to reflect that.